integrations/salesforce-dmp/lib/index.js

```diff
@@ -137,22 +139,24 @@ SalesforceDMP.prototype.identifyV2 = SalesforceDMP.prototype.trackV2 = function
     resetDataLayer = createDataLayer()
   }
 
-  if (this.options.sendEventNames && !window.kruxDataLayer['event_name']) {
-    var name
-    if (type === 'track') {
-      name = msg.event()
-    } else {
-      if (this.options.sendIdentifyAsEventName) {
-        name = type
+  if (window.kruxDataLayer) {
```

Comment:
The diff GitHub is showing here is just from indenting the code because of this `if`, right?